packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -24,11 +24,11 @@ class WebpackCLI {
                 ),
         });
     }
-
     async tryRequireThenImport(module, handleError = true) {
         let result;
 
         try {
+            delete require.cache[module];
```

Comment:
Unsafe and very bad for perf

Reply:
its cache for one file, perf wont change that much because of this